storageClient/storageClient.go

```diff
@@ -102,6 +104,18 @@ func (c *StorageClient) ListGroups(ctx context.Context, params ListGroupsRequest
 	return result, err
 }
 
+// AddMembersToGroup Adds Clients to a group and returns the succesfuly added clients (if any).
+func (c *StorageClient) AddMembersToGroup(ctx context.Context, params AddToGroupRequest) (*AddToGroupResponse, error) {
```

Comment:
`AddGroupMembers` for the method name please, `AddGroupMembersRequest` for the params, and `*[]GroupMember` for the first return value